cmd/writePipelineEnv.go

```diff
@@ -66,7 +69,7 @@ func runWritePipelineEnv(config *artifactPrepareVersionOptions) error {
 	}
 
 	// try to decrypt
-	if config.Password != "" && orchestrator.DetectOrchestrator() == orchestrator.GitHubActions {
+	if encryptedCPE && stepConfigPassword != "" {
```

Comment:
What will happen if the password is unavailable? I think the step will fail, it'd be good to have a meaningful error that indicates that the password is missing.